client/client.go

```diff
@@ -638,11 +642,26 @@ func (c *client) GetTS(ctx context.Context) (physical int64, logical int64, err
 	return resp.Wait()
 }
 
+func (c *client) GetLastTS(ctx context.Context) (physical int64, logical int64, err error) {
+	return c.GetLastLocalTS(ctx, globalDCLocation)
+}
+
 func (c *client) GetLocalTS(ctx context.Context, dcLocation string) (physical int64, logical int64, err error) {
 	resp := c.GetLocalTSAsync(ctx, dcLocation)
 	return resp.Wait()
 }
 
+func (c *client) GetLastLocalTS(ctx context.Context, dcLocation string) (int64, int64, error) {
+	lastTSOInterface, ok := c.lastTSMap.Load(dcLocation)
```

Comment:
According to https://github.com/pingcap/tidb/issues/21522 , we also need to maintain the arrival timestamp in lastTSMap.

Reply:
Thanks for your review, I agree with you.